packages/drawer/src/react/__specs__/index.spec.tsx

```diff
@@ -1,46 +1,52 @@
-import React from 'react'
+import React, { useState } from 'react'
 import { fireEvent, render } from '@testing-library/react'
 
 import Drawer from '..'
 
 describe('Drawer', () => {
-  describe('when the arrow icons is clicked', () => {
-    it('toggles open/close', () => {
-      const { container } = render(
-        <Drawer base={<div />}>
-          <div />
+  describe('when uncontrolled', () => {
+    it('toggles open/close', async () => {
+      const { getByTestId } = render(
+        <Drawer>
+          <Drawer.Summary data-testid="head">
```

Comment:
Not a huge deal in this context, but perhaps helpful generally:  I find it helpful to not create new names for things, creating the need to mentally track synonyms. In this case, if the testid is on the Drawer.Summary, it could be called "summary" instead of "head", and on line 20, one would know more easily what's being referenced on line 11. One might make the same argument for "btn" on line 20.

Reply:
forgot to refactor that